src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -102,3 +103,28 @@
                         :alignment   :flex-start}
     :description       :text
     :description-props {:text (string/replace constants/path-default-wallet #"/" " / ")}}])
+
+(def network-names [:ethereum :optimism :arbitrum])
+
+(def address "0x39cf6E0Ba4C4530735616e1Ee7ff5FbCB726fBd4")
+
+(def data-item-state
+  {:blur?               false
```

Comment:
curious why we have `blur?` in temporary? this seems like a configuration of the component and something we already know. - seems like we should have a function to map through the mock-data + the ui presets

Reply:
@J-Son89 `blur?` was redundant here, removed it.